### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by simond10** [simond10/Momco-Android#1]

**Comment by danielyang112** [comment]
hi, what's the problem here?

**Comment by simond10** [comment]
Hi Daniel.How are you?

**Comment by danielyang112** [comment]
Hi Simon, i am good. Long time no see. hehe

**Comment by simond10** [comment]
same here :)

**Comment by danielyang112** [comment]
missing you

**Comment by danielyang112** [comment]
congratulations!!!!!

**Comment by simond10** [comment]
me, tooooo .Thanks for your gift on my wedding ~

**Comment by danielyang112** [comment]
no thanks.. is everyone good from your family?

**Comment by simond10** [comment]
yes.How are you doing these days?

**Comment by danielyang112** [comment]
just like a pig. you know our life well. i really envy you~

**Comment by simond10** [comment]
:) you don't need to envy me.I'm still in three-two-four.lol

**Comment by danielyang112** [comment]
kk.. are you working on 3.24?

**Comment by simond10** [comment]
yes

**Comment by simond10** [comment]
Btw did you get a chance to write a letter to your mom?
I didn't call because it makes your mom miss you more and more

**Comment by simond10** [comment]
If you have sth to tell your mom or ask urgently, I can do that for you ~

**Comment by simond10** [comment]
good.let me know if you have urgent thing with your family.I can do it in a day :)

**Comment by danielyang112** [comment]
great. i will let you know if i have one. btw, how long will you work there?

**Comment by simond10** [comment]
not sure ~ but need to be here for sometime.Not bad here :)

**Comment by simond10** [comment]
what is your weight?

**Comment by danielyang112** [comment]
i see.. kk it's still same. 77 or 78. are you more slimmer now?

**Comment by simond10** [comment]
no.around 74~75

**Comment by danielyang112** [comment]
oh.. sounds good. she might be doing good to you~

**Comment by simond10** [comment]
:) how's your jh?

**Comment by simond10** [comment]
wrote a letter to her?

**Comment by danielyang112** [comment]
yeah, i did, but she didn't answer to it. so i decided to forget about her.
actually i am no luck with girls

**Comment by danielyang112** [comment]
did you tell ryeoah about me?

**Comment by simond10** [comment]
sad to hear that.no replies? I can't understand it

**Comment by simond10** [comment]
:) no connection with her but I saw her number in my wife's contact list

**Comment by danielyang112** [comment]
someone called her to come and receive the letter and there was only one day to write a reply, but she said she's busy

**Comment by danielyang112** [comment]
it also happened when jonggun tried to contact her. which means she answered me in indirect way

**Comment by simond10** [comment]
I see.Maybe you're not doing well at that time.....

**Comment by simond10** [comment]
Cheer up, my brother.You're still young and strong ~

**Comment by danielyang112** [comment]
i had some difficult time for last couple of months after the vacation and it's hard to tell you here.
but finally, i decided to forgive up about love and life and just live like a machine here

**Comment by danielyang112** [comment]
thanks, my bro~

**Comment by danielyang112** [comment]
do you have any interesting news?

**Comment by simond10** [comment]
I see.Let's meet and talk in detail when you're back

**Comment by danielyang112** [comment]
of course. hehe

**Comment by simond10** [comment]
no interesting news ~ .still in same place ~

**Comment by danielyang112** [comment]
i see. are all indians doing freelancing?

**Comment by simond10** [comment]
no.only 1 guy is working with me and rest of all are in hospital and one is not coder anymore ~

**Comment by danielyang112** [comment]
who's not coder anymore? hosong from 2nd?

**Comment by simond10** [comment]
yes

**Comment by danielyang112** [comment]
what's he doing?

**Comment by simond10** [comment]
works for the S.H company

**Comment by simond10** [comment]
buwon

**Comment by danielyang112** [comment]
oops..

**Comment by danielyang112** [comment]
why don't you become buwon like him?

**Comment by simond10** [comment]
... I have no way ...

**Comment by danielyang112** [comment]
you're much better than him..

**Comment by danielyang112** [comment]
anyway, what's your monthly amount to earn? i am gonna send it through paypal or freelancer

**Comment by simond10** [comment]
Thanks but finally he is better than me now

**Comment by danielyang112** [comment]
don't say "as much as you can". kkk

**Comment by simond10** [comment]
haha.How's your business these days?
I contacted with our guys in China but they also are using chinese freelancer and can't do transaction to china account and it's team account ~

**Comment by danielyang112** [comment]
not great, but i can do anything for you~
we are using malaysian freelancer account and one guy did same to jang several months ago.
so it must be possible

**Comment by danielyang112** [comment]
also i can send money directly through paypal

**Comment by simond10** [comment]
Thanks.I don't want paypal transaction because they can catch our transaction :)
Freelancer is better I guess

**Comment by simond10** [comment]
five hundred is enough for  me per a month ~
But I hope this doesn't make much effect on your quota

**Comment by danielyang112** [comment]
sure. no problem. do you need it this week? or is it okay to do it on next week?

**Comment by simond10** [comment]
which one is better for you?

**Comment by danielyang112** [comment]
kk, i will send 500 every month and don't lose your weight because of doing this..

**Comment by simond10** [comment]
awesome.You're my brother !!!

**Comment by danielyang112** [comment]
it's already friday and you know we don't work after 1pm, so what about doing it next week?

**Comment by danielyang112** [comment]
are you urgent?

**Comment by simond10** [comment]
so you will use same account as last time?

**Comment by simond10** [comment]
no problem

**Comment by danielyang112** [comment]
cool.. when do you come online usually?

**Comment by simond10** [comment]
btw you don't work after 1pm from friday?

**Comment by danielyang112** [comment]
1pm~7pm(very happy sleeping time)

**Comment by danielyang112** [comment]
you're not working at night, are you?

**Comment by simond10** [comment]
haha ~.I guess your place is the best one ~

**Comment by simond10** [comment]
sure.I will go home anytime when I wish :)

**Comment by simond10** [comment]
yes.this is the only way

**Comment by danielyang112** [comment]
i see.. actually i'd like to send more, but i am not doing well these days..

**Comment by simond10** [comment]
oh no need to do that.that amount is surely enough for me~

**Comment by simond10** [comment]
I'm not sure you can remember this idiom ~ "It's the thought that counts".

**Comment by danielyang112** [comment]
kk how many bottles of oil do you get from there?

**Comment by simond10** [comment]
1 bottle for 1 month or 2 months~

**Comment by danielyang112** [comment]
sorry, i have bad memory.. my memory only works well on girls

**Comment by simond10** [comment]
kk.so do I.I surely feel my memory is not going well
Don't drink too much ~

**Comment by simond10** [comment]
and I don't smoke ~

**Comment by danielyang112** [comment]
you have a camera in your head.. you know people here don't drink much and it's not funny to drink alone

**Comment by danielyang112** [comment]
you shouldn't.. you're not alone~~~

**Comment by simond10** [comment]
:)

**Comment by simond10** [comment]
yes.You also need to prepare from now on ~ .I know you and the work there and it made us drink and smoke but don't do it too much

**Comment by danielyang112** [comment]
kk i see what you mean.. when i was at vacation, i had chance to do it, but my little finger didn't stand~

**Comment by simond10** [comment]
I remembered it :)

**Comment by simond10** [comment]
so will you create a job now and hire me next week? Is this more natural way?

**Comment by danielyang112** [comment]
yes, we will set it up next week

**Comment by danielyang112** [comment]
i am not managing the account now, so i can't do it today

**Comment by simond10** [comment]
I see.I will be available here on Monday morning .

**Comment by danielyang112** [comment]
ok, sounds good

**Comment by danielyang112** [comment]
where did you buy your home?

**Comment by simond10** [comment]
can't remember exactly but the early of july before my wedding ~

**Comment by simond10** [comment]
sorry.pyongchon

**Comment by danielyang112** [comment]
i see.. can i ask how much it was?

**Comment by danielyang112** [comment]
just need to figure out how much i should prepare for my honeyhome

**Comment by simond10** [comment]
haha.not too much.You can buy the same house when you come back

**Comment by danielyang112** [comment]
3 rooms?

**Comment by simond10** [comment]
yes 3 rooms.I live with mother-in-law

**Comment by danielyang112** [comment]
oh.. really? why? they are living in central district, aren't they?

**Comment by simond10** [comment]
mom is sick and is living with me now.so so so

**Comment by danielyang112** [comment]
i see.. sorry to hear that.  how's married life? do you recommend it to me?

**Comment by simond10** [comment]
haha.Yes.Good life.say "come back early' in the morning and say "hi" when I come back ...

**Comment by danielyang112** [comment]
lol i am not sure when i can hear that lovely words~ i might become a ghost here~

**Comment by danielyang112** [comment]
oh.. how's songil doing? he might get married soon because hyoryon returned home few weeks ago

**Comment by simond10** [comment]
don't worry.You will hear the more lovely message soon.
You have many girls ...

**Comment by simond10** [comment]
I also heard about it :) You told me he did something wrong there but there seems to be no restriction or suspension on his way ~

**Comment by danielyang112** [comment]
they are all same.. no one can blame another

**Comment by simond10** [comment]
I guess so ...
Anyway the girls might be taller than him ~

**Comment by danielyang112** [comment]
kk.. i am afraid to see such pictures~ is it possible to send your wedding photo?

**Comment by simond10** [comment]
haha.no way.this is the only way for us to talk.and we have the most power and strong securing system

**Comment by danielyang112** [comment]
i see... hmm.. do you know how bang's doing?

**Comment by simond10** [comment]
he had gone to another place.not in here anymore

**Comment by danielyang112** [comment]
guys in hospital are looking for chance to escape?

**Comment by simond10** [comment]
yes and some are waiting to go for travel

**Comment by simond10** [comment]
But it's still not easy to escape

**Comment by danielyang112** [comment]
i heard delegation is also not easy these days and is that so?

**Comment by simond10** [comment]
I heard it but it doesn't seems to be difficult.Many of them are preparing for travel here

**Comment by danielyang112** [comment]
don't forget to introduce good girls to me, especially ryoah

**Comment by simond10** [comment]
haha.Let me try but I'm not sure she can wait for you by that time ~

**Comment by danielyang112** [comment]
kk.. just let her know that i am still thinking about her.. i am just curious whether she remembers me or not

**Comment by simond10** [comment]
:) I'm sure she still remembers you .You were the star ~

**Comment by simond10** [comment]
one thing.when I visit https://github.com/, I still can see the all comments between you and me
Can you find how to delete them?not easy to search the way here

**Comment by danielyang112** [comment]
i am collaborator and i can't see that option, but i can see x buttons on each messages

**Comment by simond10** [comment]
I see.let me try

**Comment by simond10** [comment]
How about my friend there?

**Comment by danielyang112** [comment]
i don't see comments here, but i see it on github landing page

**Comment by simond10** [comment]
that's what I mean.

**Comment by danielyang112** [comment]
i can't figure it out...

**Comment by danielyang112** [comment]
you can see only my comments, right?

**Comment by simond10** [comment]
:) it will be disappeared after some hours or days I guess

**Comment by simond10** [comment]
yes

**Comment by danielyang112** [comment]
i see :) do you often go to restaurants?

**Comment by simond10** [comment]
not much.I need to stay home with wife :)

**Comment by simond10** [comment]
did you think about me on my birthday?

**Comment by danielyang112** [comment]
of course... we went shopping first time after vacation and i thought about you

**Comment by simond10** [comment]
oh, no shopping until that time after vacation?

**Comment by danielyang112** [comment]
yes.. once in two or three months, it's getting longer

**Comment by simond10** [comment]
I know only the winner for 6 months can go to restaurant ~

**Comment by danielyang112** [comment]
oh really? we are not interested at all about restaurant.. you know the girls here
we didn't go there on 10.10 even we had chance to go

**Comment by simond10** [comment]
haha.you don't miss SR?

**Comment by danielyang112** [comment]
not at all.. i decide to be cool to girls and she's not my type~

**Comment by simond10** [comment]
so cool! Anyway I miss you ...

**Comment by danielyang112** [comment]
me too... i miss you a lot~ don't say to your wife that i miss you.. she might misunderstand us

**Comment by simond10** [comment]
haha.that also makes sense ~

**Comment by danielyang112** [comment]
did you explain her about me that i have no relation to jangmi?

**Comment by danielyang112** [comment]
does she still hate my name?

**Comment by simond10** [comment]
no.I can't do that.You must have relation with her
I'm trying to convince her that "he had relation with her and ... but he is my lovely brother so you need to treat him well " :)

**Comment by danielyang112** [comment]
haha.. no problem for me if she doesn't hate me~ i am just afraid she doesn't give me wine when i visit your home

**Comment by simond10** [comment]
haha.Don't worry.I will handle it

**Comment by danielyang112** [comment]
otherwise you can call me another name, "Daniel"

**Comment by simond10** [comment]
:) l already showed her your pictue and told your name but the reation wasn't good
She told me you're not bad ~

**Comment by simond10** [comment]
*reaction
*your picture

**Comment by danielyang112** [comment]
the reaction wasn't good because she heard my name?

**Comment by simond10** [comment]
you remember that we talked about our team is the most handsome team in asia.lol

**Comment by simond10** [comment]
no.by your face only

**Comment by danielyang112** [comment]
kk.. did you tell her about me and ryoah's relationship?

**Comment by simond10** [comment]
no.she doesn't like ryoah

**Comment by danielyang112** [comment]
aha.. now i understand~ kk so it's not possible to connect with her

**Comment by simond10** [comment]
but now she doesn't concern about jm anymore

**Comment by simond10** [comment]
if you wish, I can let you know her number only :)
But you have skills and you can do anyting .lol

**Comment by danielyang112** [comment]
of course she does~ you're totally hers now. i thought i had skills, but it's just for making fun, not for eternal love~ all of girls around me betrayed or left me

**Comment by simond10** [comment]
Have a confidence.You were not your match

**Comment by simond10** [comment]
*she wasn't

**Comment by danielyang112** [comment]
i heard you had wedding in hh, not py and is there special reason for that?

**Comment by simond10** [comment]
nothing.she also was born in hh

**Comment by simond10** [comment]
ok.I need to go for  a lunch now
I will be availalbe on Monday morning.
Have a nice weekend and talk to you later ~

**Comment by danielyang112** [comment]
sure. enjoy your lunch and see you on monday
have a great weekend!

**Comment by simond10** [comment]
Hi Daniel
Are you here?

**Comment by danielyang112** [comment]
Hi, good morning

**Comment by simond10** [comment]
Good moring.How's your weekend?

**Comment by danielyang112** [comment]
just as usual~ what's your freelancer id?

**Comment by simond10** [comment]
I will let you know but create an anonymous project and I will bid on it and after some hours or so you can hire me

**Comment by simond10** [comment]
we don't want the title like "project for ..." :)

**Comment by danielyang112** [comment]
what about directly hiring you on your profile page?

**Comment by simond10** [comment]
Not bad but I don't want the project title contatins my id ...
Btw you don't think freelancer doubt that transaction?

**Comment by danielyang112** [comment]
i see, let me create one for you

**Comment by simond10** [comment]
thanks.let me know the project title then

**Comment by danielyang112** [comment]
Android Intercom  expert needed

**Comment by danielyang112** [comment]
did you bid on it?

**Comment by simond10** [comment]
I found that project but it's still loading.very slow connection here ...

**Comment by simond10** [comment]
I bid on that project with good proposal :)

**Comment by danielyang112** [comment]
good. what's your id?

**Comment by simond10** [comment]
realmgpro : 3rd place .lol

**Comment by danielyang112** [comment]
i sent you a message on freelancer

**Comment by danielyang112** [comment]
someone already did the same on this account 3 weeks ago

**Comment by danielyang112** [comment]
maybe it's jong~ so we should know each other on freelancer

**Comment by simond10** [comment]
I see.I'm on freelancer now :)

**Comment by danielyang112** [comment]
let me know when you want to release it

**Comment by simond10** [comment]
thanks.I will let you 
Have a good day ~

**Comment by danielyang112** [comment]
you too~

**Comment by simond10** [comment]
Hi Daniel.How are you?

**Comment by danielyang112** [comment]
Good. How are you?

**Comment by simond10** [comment]
sounds good.I'm fine
Will you release the payment today?

**Comment by danielyang112** [comment]
just released and left feedback

**Comment by simond10** [comment]
Thanks
You saved my honor :)

**Comment by danielyang112** [comment]
lol.. my pleasure~

**Comment by simond10** [comment]
Btw have you ever heard about next gf of kyh after sy?

**Comment by danielyang112** [comment]
i heard she's from sangdae, why do you ask that?

**Comment by simond10** [comment]
oh, I've never heard of her I remember ...

**Comment by danielyang112** [comment]
i heard it recently after i told him about jh

**Comment by simond10** [comment]
He is liar... They must be very close and were born on same date

**Comment by danielyang112** [comment]
oh.. that's destiny, he only told me he met her, but not seriously

**Comment by simond10** [comment]
They met on their birthday and there was a letter or so between them ...

**Comment by danielyang112** [comment]
oh.. lettter~ lucky guy.. he's better than me and fast like a rabbit, hihi

**Comment by danielyang112** [comment]
i never received a letter from girls, except bang

**Comment by simond10** [comment]
lol ... I saw her recently and seems like she is kind of old now

**Comment by danielyang112** [comment]
yeah, she looks much older than before

**Comment by simond10** [comment]
She must be still missing you and your skills ... :)

**Comment by danielyang112** [comment]
haha, not at all~ she might have found another love.

**Comment by simond10** [comment]
maybe yes but she can't find a skilful man like you .lol

**Comment by danielyang112** [comment]
it doesn't work now and i feel bit sorry for her~

**Comment by danielyang112** [comment]
she was the only girl who really liked me and waited for me

**Comment by simond10** [comment]
That also make sense but it's impossible ~

**Comment by simond10** [comment]
so you don't think jh will wait for you?

**Comment by danielyang112** [comment]
she won't get married before i return, but she's not actually waiting for me

**Comment by simond10** [comment]
... if yes, that's not yours.
As an example, the girl betrayed kks(1th) but he got better girl now 
don't worry

**Comment by danielyang112** [comment]
yeah, i heard that~ actually jh didn't betray me and we were just nothing

**Comment by simond10** [comment]
haha.I didn't mean that girl.

**Comment by danielyang112** [comment]
kk, i know

**Comment by simond10** [comment]
regarding jh, you like her and your family like her also and two families are close
There is no reason why you can't get married ...

**Comment by danielyang112** [comment]
the biggest reason is i don't like her, my heard doesn't bounce when i think of her

**Comment by simond10** [comment]
... I see.You will come back on 2018? and you must be 28.still young

**Comment by danielyang112** [comment]
maybe~ btw, when you expect your prince?

**Comment by simond10** [comment]
haha.didn't start yet ...

**Comment by simond10** [comment]
but must be born next year ~

**Comment by simond10** [comment]
you told me you prefer a girl?still same thought?

**Comment by danielyang112** [comment]
yeah~ i like a daughter

**Comment by danielyang112** [comment]
sorry, i will be back soon

**Comment by simond10** [comment]
ok

**Comment by AlifKuan** [comment]
I will leave for now.see you later ~

**Comment by simond10** [comment]
Hi.I will be unavailable here from now on.
Thanks for your help
Good luck~
